packages/@expo/config-plugins/src/android/Updates.ts

```diff
@@ -1,8 +1,11 @@
+import { ExpoConfig } from '@expo/config-types';
+import { run } from 'envinfo';
```

Comment:
Is this needed? If so be sure to add to pkg json

Reply:
Removed these